__tests__/index.test.js

```diff
@@ -340,6 +340,40 @@ describe('GET /api/articles/:article_id (comment_count)',()=>{
 
 })
 
+describe('GET /api/articles (topic query',()=>{
+    test('return a 200 code when a request is made to the api',()=>{
+        return request(app)
+        .get('/api/articles?topic=mitch')
```

Comment:
As per comments above, you can delete this first test as you are asserting for this in the test below 👀